src/mscorlib/src/System/Globalization/TimeSpanFormat.cs

```diff
@@ -36,6 +36,29 @@ internal enum Pattern
         // 
         internal static String Format(TimeSpan value, String format, IFormatProvider formatProvider)
```

Comment:
It maybe useful to move TimeSpanFormat.cs and TimeSpanParse.cs to shared first so this change ports to CoreRT for free. The CoreCLR and CoreRT copies are not that far off.

Reply:
Ugh, somehow I managed to convince myself that all of the files were already shared :smile:

Reply:
I've added some commits to move the files to shared.